packages/terra-avatar/src/variants/Generic.jsx

```diff
@@ -31,45 +35,56 @@ const propTypes = {
    * Overrides the default size.
    */
   size: PropTypes.string,
+  /**
+   * Sets the Generic Avatar type to One of the following variants `single-user`, `shared-user`, or `provider`.
+   */
+  variant: PropTypes.oneOf([GENERIC_VARIANTS.SINGLE_USER, GENERIC_VARIANTS.SHARED_USER, GENERIC_VARIANTS.PROVIDER]),
```

Comment:
@supreethmr we will have to export it separately so that consumer can access it.